docs/terminology.md

```diff
@@ -0,0 +1,143 @@
+# Thentos Terminology
+
+## Introduction
+
+Let's start with explaining the obvious.
+
+Users, names, roles, groups, persmissions, etc. are names for rather
+abstract concepts.  As it goes with abstraction, having a good
+intuition of their practical meaning can be hard.  Different software
+systems use different abstractions and thus use the same names with
+different meaning without always being aware of the differences.
+
+This document is attempt to establish common terminology and make it
+accessible for everybody who gets in contact with thentos.
+
+Unless stated otherwise, all terms relate to concepts as implemented
+in thentos.  Having that said, as
+https://github.com/liqd/adhocracy3.mercator is the first web app to
+use thentos, we are covering the terminology for that, too.  Other web
+apps will have different terms and concepts, and this document may be
+extended in the future to cover more of them.
+
+
+### User
+
+A (thentos) user corresponds to a real person using thentos and
+associated web apps.
+
+It contains a login name, password, and possibly other data like email
+or residential address (more likely) or payment information in various
+payment systems (currently less likely).
+
+Interactions between the real person and thentos or a web app are
+called actions.  An attempted action may trigger an authorization
+error (see below).
+
+
+### Nym
+
+FIXME: used to be called persona in thentos, too.  fisx likes 'nym'
+better: less overloading, shorter, prettier.  should we have a vote on
+renaming it?  other options: pseudonym, nick, identity.
+
+A person may want to assume different identities at different times.
+A simple way to achieve that would be to create many users, but on the
+other hand some things like residential address or system
+notifications, should be shared between those identities.
```

Comment:
add a sentence along the lines of: "this is a technical concept.  there are different UI alternatives, e.g., ..., of which the technical concept is independent."